lib/generators/bigbluebutton_rails/templates/migration_3_1_0_b.rb

```diff
@@ -0,0 +1,11 @@
+class BigbluebuttonRailsTo310B < ActiveRecord::Migration
+  def up
+    add_column :bigbluebutton_meetings, :internal_meeting_id, :string
+    drop_table :bigbluebutton_server_configs
+    drop_table :bigbluebutton_room_options
+  end
+
+  def down
+    remove_column :bigbluebutton_meetings, :internal_meeting_id
```

Comment:
Remove the `raise ActiveRecord::IrreversibleMigration, "Can't undo due to loss of values during migration"` on `down` method of `migration_3_1_0.rb`.